pkg/lang/javascript/nestjs.go

```diff
@@ -122,6 +122,7 @@ func (p *NestJsHandler) handleFile(f *core.SourceFile, unit *core.ExecutionUnit)
 
 		actedOn, newfileContent := p.actOnAnnotation(f, &listen, fileContent, appName, p.Config.GetResourceType(unit), cap.ID)
 		if actedOn {
+			annot.Detach()
```

Comment:
Shouldn't this only detach if the code was commented out? That doesn't always happen if actedOn I don't think (eg, EKS case)

Reply:
You're right. I'll move this to the lambda section in `actOnAnnotation` for both JS expose plugins.